src/app/registration/facility-select/facility-select.component.html

```diff
@@ -110,7 +110,7 @@
               Pass availability -
               <span [ngClass]="timeConfig.PM.text">{{ timeConfig.PM.text }}</span>
             </p>
-            <p class="card-text">Arrive after 1pm</p>
+            <p class="card-text" data-testid="arrive-departure-text-PM">Arrive after 1pm</p>
```

Comment:
I'm not sure how I feel about the idea of adding code in here exclusively to provide an element identifier for testing - it works but in the future we may have to find a better way to parse through blocks of html for testing.

Reply:
@wilwong89 @cameronpettit you can do something like this in your spec:

```
const elementThing = fixture.debugElement.query(By.css('#idgoeshere'))
```

This way we don't pollute the `data-testid` attribute all over the place and instead just use `id='idgoeshere'`